### PR TITLE
Component selection

### PR DESCRIPTION
**Pull request by ykulbak**
Should be tested with Solibri and bimsync

### CODE REVIEW

**Review by cschladetsch — COMMENTED** [review]
Punctuation at end of paragraphs and bullet points is inconsistent.

There is unneeded vertical white space.

**Comment by leosobral** [comment]
Tested OK in Bimsync

File: Test Cases/v3.0/Visualization/Component selection/unzipped/bcf.version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version VersionId="3.0">
+  <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Component selection/unzipped/bcf.version:3]: Remove detailed version

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1
Imports fine in Solibri. Detailed version needs to go.

**Comment by ykulbak** [comment]
@pasi-paasiala @jasollien ready for review

**Review by jasollien — APPROVED** [review]

**Pull request merged**